packages/reports/addon/components/filter-values/time-dimension/base.ts

```diff
@@ -23,16 +23,11 @@ export default class BaseTimeDimensionFilter extends Component<TimeDimensionFilt
   @readOnly('args.filter.parameters.grain') dateTimePeriod!: Grain;
 
   /**
-   * the active dateTimePeriod which does not include 'all'
+   * the active dateTimePeriod
    */
   @computed('dateTimePeriod')
   get calendarDateTimePeriod() {
```

Comment:
Do we need this property anymore, can we just use the arg?

Reply:
I was planning on keeping this for now because I think it could be useful for sub day grains